README.md

```diff
@@ -93,5 +93,7 @@ ios_application(
 See the [tests](https://github.com/bazel-ios/rules_ios/tree/master/tests)
 directory for sample applications.
 
-## Special note about debugging
+## Special notes about debugging xcode projects
+Note that when generating xcodeprojects, it is necessary to supply the `--//rules:local_debug_options_enabled` so that debugging of swift files will work.
```

Comment:
```suggestion
Note that when generating Xcode projects, it is necessary to supply the `--//rules:local_debug_options_enabled` so that debugging of swift files will work.
```

also, should include the repository in the flag name